hexrd/ui/indexing/fit_grains_results_dialog.py

```diff
@@ -60,14 +60,17 @@ def __init__(self, data, material=None, parent=None):
         self.load_cmaps()
         self.reset_glyph_size(update_plot=False)
 
-        # Add column for equivalent strain
+        # Add columns for equivalent strain and hydrostatic strain
         eqv_strain = np.zeros(self.num_grains)
+        hydrostatic_strain = np.zeros(self.num_grains)
         for i, grain in enumerate(self.data):
             epsilon = vecMVToSymm(grain[15:21], scale=False)
             deviator = epsilon - (1/3) * np.trace(epsilon) * np.identity(3)
             eqv_strain[i] = 2 * np.sqrt(np.sum(deviator**2)) / 3
-        # Reshape so we can hstack it
+            hydrostatic_strain[i] = 1 / 3 * np.trace(epsilon)
```

Comment:
@joelvbernier I'm double-checking the math. Is this right? hydrostatic strain is `1 / 3 * np.trace(epsilon)`, and hydrostatic stress is `1 / 3 * np.trace(sigma)`?

Reply:
Resolved by https://github.com/HEXRD/hexrdgui/pull/690#issuecomment-755519235